<ui version="4.0"> 
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Civilization</string>
  </property>
    <property name="windowIcon">
   <iconset>./res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" />
    <property name="autoFillBackground">
      <bool>true</bool>
    </property>
  <widget class="QStackedWidget" name="stackedWidget">
    <property name="autoFillBackground">
      <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>640</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_1">
      <widget class="QLabel" name="start_movie">
        <property name="geometry">
          <rect>
            <x>0</x>
            <y>0</y>
            <width>960</width>
            <height>640</height>
          </rect>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
      </widget>
      <widget class="QLabel" name="button_background">
        <property name="geometry">
          <rect>
            <x>400</x>
            <y>340</y>
            <width>209</width>
            <height>262</height>
          </rect>
        </property>
        <property name="styleSheet">
          <string notr="true"> background-color: rgb(255, 255, 255,0);</string>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
      </widget>
      <widget class="QLabel" name="label">
        <property name="geometry">
          <rect>
            <x>850</x>
            <y>615</y>
            <width>100</width>
            <height>20</height>
          </rect>
        </property>
        <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);font: 500 18pt &quot;m5x7&quot;</string>
        </property>
        <property name="text">
          <string>v0.0.0-alpha</string>
        </property>
      </widget>
      <widget class="QPushButton" name="B_start">
        <property name="geometry">
          <rect>
            <x>401</x>
            <y>350</y>
            <width>200</width>
            <height>80</height>
          </rect>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
        <property name="styleSheet">
          <string notr="true"> background-color: rgb(255, 255, 255,0);</string>
        </property>
        <property name="text">
          <string/>
        </property>
      </widget>
      <widget class="QPushButton" name="B_set">
        <property name="geometry">
          <rect>
            <x>401</x>
            <y>431</y>
            <width>200</width>
            <height>80</height>
          </rect>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
        <property name="styleSheet">
          <string notr="true"> background-color: rgb(255, 255, 255,0);</string>
        </property>
        <property name="text">
          <string/>
        </property>
      </widget>
      <widget class="QPushButton" name="B_quit">
        <property name="geometry">
          <rect>
            <x>401</x>
            <y>512</y>
            <width>200</width>
            <height>80</height>
          </rect>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
        <property name="styleSheet">
          <string notr="true"> background-color: rgb(255, 255, 255,0);</string>
        </property>
        <property name="text">
          <string/>
        </property>
      </widget>
    </widget>
    <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="setting_movie">
        <property name="geometry">
          <rect>
            <x>0</x>
            <y>0</y>
            <width>960</width>
            <height>640</height>
          </rect>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
      </widget>
      <widget class="QLabel" name="setting_background">
        <property name="geometry">
          <rect>
            <x>20</x>
            <y>20</y>
            <width>920</width>
            <height>600</height>
          </rect>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
        <property name="styleSheet">
          <string notr="true"> background-color: rgb(255, 255, 255,0);</string>
        </property>
      </widget>
      <widget class="QPushButton" name="B_setreturn">
        <property name="geometry">
          <rect>
            <x>70</x>
            <y>70</y>
            <width>50</width>
            <height>50</height>
          </rect>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
        <property name="icon">
          <iconset>
            <normaloff>./res/start/1.png</normaloff>./res/start/1.png
          </iconset>
        </property>
        <property name="iconSize">
          <size>
            <width>32</width>
            <height>32</height>
          </size>
        </property>
        <property name="styleSheet">
          <string notr="true"> background-color: rgb(255, 255, 255,0);</string>
        </property>
        <property name="text">
          <string/>
        </property>
      </widget>
    </widget>
   </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
